js/plugins/configButton.js

```diff
@@ -0,0 +1,39 @@
+var configCheckManager = new clinicalTimelinePlugin("configButton", null, {configDivId: "#timeline-wrapper #config-div"});
```

Comment:
Here you are using the names of certain elements on the page directly. How would I add this plugin on cBioPortal w/o adding these elements to the page?